spec/inputs/elasticsearch_spec.rb

```diff
@@ -408,45 +408,6 @@ def synchronize_method!(object, method_name)
           end
         end
       end
-
```

Comment:
Do the currently available tests cover use case you are fixing by this PR already? If not, lets add tests that cover that scenario specifically and ensure it wont regress if changed.